Dockerfile.konflux

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 IBM Corporation
```

Comment:
can we rename it to Containerfile?

Reply:
is the license required?

Reply:
License removed, We trying to keep standardized naming across all components as `Dockerfile.konflux`.